src/DataConverter/AbstractConverter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Shaarli Netscape Bookmark Parser package.
+ *
+ * (c) "Matthias Morin" <mat@tangoman.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Shaarli\NetscapeBookmarkParser\DataConverter;
+
+/**
+ * @author "Matthias Morin" <mat@tangoman.io>
+ */
+abstract class AbstractConverter implements DataConverterInterface
+{
+    public const FORMAT = '';
```

Comment:
Just to avoid having a valid empty `FORMAT` in sub classes:

```suggestion
    public const FORMAT = null;
```